include/elliptics/interface.h

```diff
@@ -400,6 +400,9 @@ struct dnet_config
 
 	/* so that we do not change major version frequently */
 	int			reserved_for_future_use[8];
+
+	unsigned int            grpc_thread_num;
+	const char	        *grpc_address;
```

Comment:
It is public header, try to keep it backward compatible as long as possible

Reply:
Please, put `grpc_address` into `dnet_config_data` as it is done for other addresses.

Reply:
123ea1f